### ISSUE TITLE
Loss is not connected to the model

### ISSUE THREAD

**Issue by asash** [suchanadatta/qppBERT-PL#7]
Hi, in the training code your loss is not connected to the model:
https://github.com/suchanadatta/qppBERT-PL/blob/d706122b5da25d9600d3912c00a2b253bf914eaa/qpp_model/train.py#L131

The model doesn't train with this disconnected loss. 

Can you please share the code you used for the paper?
